README.md

```diff
@@ -17,28 +22,64 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# **KubeTurbo** <sup><sub>_Autonomic Scheduling for Kubernetes Implementations with Turbonomic_</sub></sup>
+# **KubeTurbo** 
 
 ----
 
 1. [Overview](#overview)
 2. [Getting Started](#getting-started)
-  * [System Requirements](#system-requirements)
+3. [Kubeturbo Use Cases](#use-cases)
+4. [Coming Soon](#coming-soon)
 
 ----
 
 ## Overview 
 
-Turbonomic will discover application containers 
-running across all pods in a Kubernetes cluster. In order to add this target, you must create a Kubeturbo configuration file which includes your Turbonomic credentials,and a custom pod definition that will be used by Kubelet to create a mirror pod running the Kubeturbo service.
+Kubeturbo leverages [Turbonomic](https://turbonomic.com/) patented analysis engine to provide visibility and control across the entire stack, to assure the performance of running micro-service in Kubernetes Pods, as well as the efficiency of underlying infrastructure.
 
 ## Getting Started 
-### System Requirements 
-
+### Prerequisite
 * Turbonomic 5.9+ installation
 * Kubernetes 1.4+
-
-## Creating the Configuration File:
-[Test Link](https://github.com/DigitalCommodore/kubeturbo/blob/digicomm-anchorlinktest/deploy/general-deploy/README.md#step-one-creating-the-kubeturbo-configuration-files)
+### Kubeturbo Installation
+* [Deploy Kubeturbo](https://github.com/turbonomic/kubeturbo/tree/master/deploy)
+
+<img width="1323" alt="screen shot 2017-06-01 at 10 10 21 am" src="https://cloud.githubusercontent.com/assets/4391815/26683726/c5e06e7c-46b2-11e7-92f2-5f555fe88ea2.png">
+
+## Use Cases
+* Full-Stack Visibility by leveraging 50+ existed Turbonomic controllers, from on-prem DataCenter to major public cloud providers. No more shadow IT
+  * From Load Balancer all the way down to your physical Infrastructure
+  * Real-Time resource monitoring across entire DataCenter
+  * Real-Time Cost visibility for your public cloud deployment
+<img width="1322" alt="screen shot 2017-06-01 at 10 10 48 am" src="https://cloud.githubusercontent.com/assets/4391815/26683739/cfde3c7e-46b2-11e7-84b5-a39c12f47022.png">
+<img width="1320" alt="screen shot 2017-06-01 at 10 11 54 am" src="https://cloud.githubusercontent.com/assets/4391815/26683749/d7813a30-46b2-11e7-9283-0dbc63769d90.png">
+<img width="1403" alt="screen shot 2017-06-01 at 10 11 03 am" src="https://cloud.githubusercontent.com/assets/4391815/26683745/d3c42a38-46b2-11e7-9e6c-3587a1480afc.png">
+
+* Provide Rescheduler capability by leveraging Turbonomic analysis engine **(Execution of moving pods requires Kubeturbo to be the scheduler of the pod)**
```

Comment:
"Turbonomic's analysis engine" -or- "the Turbonomic analysis engine" (I prefer the latter)